POC/api/routes/base.py

```diff
@@ -0,0 +1,87 @@
+from fastui.events import GoToEvent
```

Comment:
fastui and basic routes like redirects and setting the fastui base route along with the api value for the backend portion of the fastui handler